.dockstore.yml

```diff
@@ -15,3 +15,6 @@ workflows:
   - subclass: WDL
     primaryDescriptorPath: /pipelines/mining/cytomining.wdl
     name: cytomining
+  - subclass: WDL
+    primaryDescriptorPath: /pipelines/test/cellpainting_workflow.wdl
+    name: cytomining
```

Comment:
Two questions here: do we want this in Dockstore?  Do we think people could run this workflow?

Also the name should probably be "cellpainting" or something

Reply:
oh totally!
`cellpainting` should be the name. 
I copied the previous 3 lines, edited the path and forgot to edit the name.
Thanks for catching it.

Reply:
Changed in latest commit